common/lib/transport/connectionmanager.js

```diff
@@ -1483,22 +1497,24 @@ var ConnectionManager = (function() {
 	};
 
 	ConnectionManager.prototype.getTransportPreference = function() {
-		return this.transportPreference || (haveWebStorage && WebStorage.get(transportPreferenceName));
+		return this.transportPreference || (this.realtime.storage.get(transportPreferenceName));
 	};
 
 	ConnectionManager.prototype.persistTransportPreference = function(transport) {
 		if(Utils.arrIn(Defaults.upgradeTransports, transport.shortName)) {
 			this.transportPreference = transport.shortName;
-			if(haveWebStorage) {
-				WebStorage.set(transportPreferenceName, transport.shortName);
+			var storage = this.realtime.storage;
+			if(storage.supportsLocal) {
+				storage.set(transportPreferenceName, transport.shortName);
 			}
```

Comment:
The result of this is that node clients persist their transport preference unnecessarily (they default to using websockets). I'd suggest either adding a conditional to persistTransportPreference for `connectionManager.transports.length > 1` (or maybe just a `persistTransportPreference` bool in the Platform object set to false for node)